src/main/resources/youtube/settings/host/values/strings.xml

```diff
@@ -1470,6 +1471,18 @@ Limitation: Automatically played feed videos will show up in your watch history.
 
 Limitation: Feed videos will play for less than 1 minute before encountering playback issues."</string>
 
+    <string name="revanced_spoof_signature_title">Spoof app signature</string>
+    <string name="revanced_spoof_signature_summary_on">App signature spoofed.</string>
+    <string name="revanced_spoof_signature_summary_off">App signature not spoofed.</string>
+    <string name="revanced_spoof_signature_user_dialog_message">"App signature will be spoofed to the original YouTube.
+
+This will only work if the following conditions are met:
+
+1. The original YouTube is installed on the device.
```

Comment:
Shouldn't it also say the original YouTube needs to be up-to-date?

So something like: `1. The original YouTube is installed and up-to-date on the device.`

Reply:
I have an old version of YouTube installed on my device, and despite this I can run a more newer version of ReVanced.

So...nope, an updated stock version isn't strictly needed.

Reply:
In the PR description alireza wrote "An official YouTube app on your phone (upgrade it to the latest version also)"

So I'm confused

Reply:
> In the PR description alireza wrote "An official YouTube app on your phone (upgrade it to the latest version also)"
> 
> So I'm confused

Maybe it's a description related to much more older versions (or a mistake), because my stock version is 19.15.35 and I can run a modded APK based on 19.24.35.

Reply:
I don't think there will be any problems even if users are using a very old version of stock YouTube
YouTube will be signed with the same key regardless of the version

Reply:
Ah I since a lot of users have left their YouTube without updates for a long time, so I added it just to make sure